CMakeLists.txt

```diff
@@ -118,6 +118,14 @@ option(WITH_GSL
 
 option(WITH_ABSEIL "Whether to use Abseil for C++latest features" OFF)
 
+if(NOT CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
+  set(OPENTELEMETRY_INSTALL_default ON)
+else()
+  set(OPENTELEMETRY_INSTALL_default OFF)
```

Comment:
(Maybe this is too late now, but) Do you think this should default to ON in all cases for backwards compatibility?

Do you think the new feature should be listed in INSTALL.md?

https://github.com/open-telemetry/opentelemetry-cpp/blob/f7026762506fd41608da323ddb170c491451393c/INSTALL.md?plain=1#L75-L86

Reply:
> (Maybe this is too late now, but) Do you think this should default to ON in all cases for backwards compatibility?

True, somehow missed this during review, and I broke one of my libraries with this change :(